staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1589,6 +1589,12 @@ message ContainerEventResponse {
 
     // ID of the sandbox container
     PodSandboxMetadata pod_sandbox_metadata = 4;
+
+    // Sandbox statuses
+    repeated PodSandboxStatus pod_sandbox_statuses = 5;
```

Comment:
how will we ensure that kubelet will work OK with the older CRI not supporting these statuses in the response?

Reply:
In general, if CRI does not support GetContainerEvents but kubelet enables evented pleg, then kubelet will update the pod status using only generic pleg but with super long relist interval. 

More specifically, if CRI somehow supports GetContainerEvents but just misses PodSandboxStatus (which I dont think will happen given that GetContainerEvents is pretty new), then it will depend on the error handling of evented pleg. IMO event_pleg should just error out if PodSandboxStatus is missing, and discard that event. In this case, all events will be discarded, and kubelet will rely only on generic pleg.